static/external-player-map.json

```diff
@@ -1,6 +1,6 @@
 [
     {
-        "name": "None",
+        "name": "Settings.External Player Settings.None",
```

Comment:
maybe a different key should be added called "translated" that is a bool value and the translating part of the code will only use the "$t" function if translated is true to avoid warnings from this PR

Reply:
Now I notice that the other names are not translated

Reply:
Another way I can think of would be 
```js
this.$te(value) ? this.$t(value) : this.$t(value)
```
[$te checks whether the translation key exists](https://kazupon.github.io/vue-i18n/api/#te)

Reply:
> Another way I can think of would be
> 
> ```js
> this.$te(value) ? this.$t(value) : value
> ```
> 
> [$te checks whether the translation key exists](https://kazupon.github.io/vue-i18n/api/#te)

that's probably the better way to do it. was not aware of the te function